### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by rajeshps** [DanFessler/react-panelgroup#4]
Hi,

When the window resizes to smaller than the defined minSize range then the following error is thrown.
Anything wrong being done on the configuration? or min size has to be dynamic?
```
[{ size: 400, minSize: 400, resize: 'dynamic' }, { size: 900, minSize: 300, resize: 'dynamic' }]
```
But the same is changed to 
```
[{ size: 400, minSize: 400, resize: 'dynamic' }, { size: 900, minSize: 0, resize: 'dynamic' }]
```
then there are no errors.
```
Uncaught RangeError: Maximum call stack size exceeded
    at Object.getPanelMinSize (PanelGroup.js:241)
    at Object.resizePanel (PanelGroup.js:201)
    at Object.resizePanel (PanelGroup.js:208)
    at Object.resizePanel (PanelGroup.js:226)
    at Object.resizePanel (PanelGroup.js:208)
    at Object.resizePanel (PanelGroup.js:226)
    at Object.resizePanel (PanelGroup.js:208)
    at Object.resizePanel (PanelGroup.js:226)
    at Object.resizePanel (PanelGroup.js:208)
    at Object.resizePanel (PanelGroup.js:226)
```

Not sure if this is expected or requires any fix.

Thanks!

**Comment by DanFessler** [comment]
Its a known edge case that I haven't solved.  If you're feeling up for it, feel free to submit a pull request

**Comment by 2blane** [comment]
This would be fixed by #15
